.github/workflows/release-vsphere-template.yaml

```diff
@@ -27,6 +27,10 @@ jobs:
             buildConfig: "basic"
           - os: "rocky 9.1"
             buildConfig: "offline"
+          - os: "oracle 9.4"
+            buildConfig: "basic"
+          - os: "oracle 9.4"
+            buildConfig: "fips"
```

Comment:
which manifests is it using? konvoy2 repo doesn't contain FIPS manifests for oracle 9.4 atm

Reply:
The FIPS overrides just tell the CLI to install the FIPS packages, then Konvoy uses the manifests to compare the installed packages vs to what it expects.

Reply:
thanks for clarifying.